software/wavelog.yml

```diff
@@ -0,0 +1,13 @@
+name: "Wavelog"
+website_url: "https://www.wavelog.org"
+source_code_url: "https://github.com/wavelog/wavelog"
+description: "Webbased Logging Software for Radio Amateurs. Enhanced QSO logging, statistics and maps for your browser."
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Docker
+tags:
+  - Miscellaneous
+demo_url: "https://demo.wavelog.org"
+related_software_url: "https://github.com/wavelog"
```

Comment:
This is usually used to provide a link to mobile/desktop clients or related application. I don't think it is very useful to point to the organizatino page, other projects listed there are not likely to be used by end users.